### PR TITLE
GP2-388 Country selection list

### PR DESCRIPTION
**Pull request by bobmeredith**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if updating requirements) Requirements have been compiled.

### CODE REVIEW

**Comment by bobmeredith** [comment]
image

**Comment by bobmeredith** [comment]
image

**Comment by bobmeredith** [comment]
image

File: core/views.py [hunk 1]
```diff
@@ -115,6 +115,11 @@ def get(self, request):
         return Response(data)
 
 
+class CountriesView(generics.GenericAPIView):
+    def get(self, request):
```
Comment (webbyfox) [inline core/views.py:119]: Just a thought, Do we need to put this endpoint behind some kind of authentication? Not sure we have any kind of authentication atm.

**Review by webbyfox — COMMENTED** [review]
@lewis-coulson-dit might want to check FE code. Just curious how do you guys do FE unit testing. I can't see any FE tests here.

**Review by dahfool — APPROVED** [review]

**Review by lewis-coulson-dit — APPROVED** [review]

**Pull request merged**